doc/performance-investigations/0001-use-grpc-for-datastore-access/summary.md

```diff
@@ -0,0 +1,28 @@
+# Use GRPC for Datastore access
+
+Use the GRPC option for Datastore access.
+
+https://github.com/ONSdigital/eq-survey-runner/tree/use-grpc-for-datastore-access
```

Comment:
These will probably end up getting deleted at some point in the future when they become stale but probably worth having a record as long as this doc exists.

Reply:
Yeah, fair point. We could prefix the branches with `performance-investigation-` to highlight they shouldn't be removed.

Reply:
👍 It's worth adding that in this doc.